arch/renesas/src/rx65n/rx65n_usbhost.c

```diff
@@ -7649,6 +7649,7 @@ static ssize_t rx65n_usbhost_transfer(struct usbhost_driver_s *drvr,
   ed->xfrinfo = NULL;
   }
   while (0);
+
 errout_with_sem:
 
   /* rx65n_usbhost_givesem(&priv->exclsem); */
```

Comment:
```suggestion
 errout_with_sem:
   /* rx65n_usbhost_givesem(&priv->exclsem); */
```

Reply:
checkpatch.sh report this warning: 
arch/renesas/src/rx65n/rx65n_usbhost.c:7654:1: error: Missing blank line before comment found